components/BookCard/BookCard.js

```diff
@@ -92,20 +96,30 @@ const StyledInfos = styled.article`
 
 const StyledImage = styled(Image)`
   margin: auto 2px;
+  border-radius: 8px;
+  box-shadow: 0 0 6px 2px rgba(0, 0, 0, 0.15);
 `;
 
 const StyledButton = styled.button`
   background-color: ${({ $isActive }) =>
-    $isActive ? "darkseagreen" : "seashell"};
+    $isActive ? "var(--color-green)" : "var(--color-light-yellow)"};
   border-radius: 4px;
+  border: none;
+  width: 32px;
+  height: 32px;
```

Comment:
Using relative units like `rem` can make your UI more accessible, as this units scale with the user's default font size settings.

```suggestion
  width: 2rem;
  height: 2rem;
```